src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <>
```

Comment:
redundant fragment

Reply:
I don't understand what the point is, if I remove these lines, the code structure breaks